clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/LoadBalancerV2UpsertHandler.groovy

```diff
@@ -60,10 +62,14 @@ class LoadBalancerV2UpsertHandler {
   }
 
   private static String modifyTargetGroupAttributes(AmazonElasticLoadBalancing loadBalancing, TargetGroup targetGroup, UpsertAmazonLoadBalancerV2Description.Attributes attributes) {
+    return modifyTargetGroupAttributes(loadBalancing, targetGroup, attributes, null)
+  }
+  private static String modifyTargetGroupAttributes(AmazonElasticLoadBalancing loadBalancing, TargetGroup targetGroup, UpsertAmazonLoadBalancerV2Description.Attributes attributes, DeployDefaults deployDefaults) {
     def targetGroupAttributes = []
     if (attributes) {
-      if (attributes.deregistrationDelay != null) {
-        targetGroupAttributes.add(new TargetGroupAttribute(key: "deregistration_delay.timeout_seconds", value: attributes.deregistrationDelay.toString()))
+      Integer deregistrationDelay = [attributes.deregistrationDelay, deployDefaults?.loadBalancing?.deregistrationDelayDefault].findResult(Closure.IDENTITY)
```

Comment:
@jrsquared I pushed one update since you approved this PR so that target groups will also use the `deployDefaults.loadBalancing.deregistrationDelayDefault`. The `modifyTargetGroupAttributes` gets called with deployDefaults on the createTargetGroup path, and with deployDefaults == null on the modifyTargetGroup path.

Reply:
👍